awx/conf/settings.py

```diff
@@ -86,49 +87,46 @@ def _ctit_db_wrapper(trans_safe=False):
             if is_atomic:
                 rollback_set = transaction.get_rollback()
                 if rollback_set:
-                    logger.debug('Obtaining database settings in spite of broken transaction.')
+                    sys.stderr.write('Obtaining database settings in spite of broken transaction.')
                     transaction.set_rollback(False)
         yield
     except DBError:
-        if 'migrate' in sys.argv and get_tower_migration_version() < '310':
```

Comment:
Don't think we need this anymore given the removal of the settings migration and the minimum version migration path.